### DISCUSSION TITLE
Syncing files to another device (Atom)

### DISCUSSION THREAD

**Discussion by MrSixty7** [community/community#22622] [New to GitHub]
<p>Hi guys</p>
<p>New to coding and new to git, so please forgive my ignorance.</p>
<p>I have been happily learning my coding using the Atom editor and I managed to get git installed and create a training repository.</p>
<p>I then wanted to sync with my second laptop, which I managed to do by trial and error pretty much.</p>
<p>I am now trying to do the same on my desktop pc but am running into an issue.</p>
<p>To sync, I installed Atom and then git. Set my global name/email in config.</p>
<p>I then opened the git gui to clone my repo from githug, pasted in the source link (from the github site, of course) and my destination folder on my local machine.</p>
<p>This is how I did it when I synced on my second laptop a few days ago, but on my desktop the cloning fails with the message… fatal: unable to access ‘<a href="https://github.com/.............git/" rel="nofollow noopener">https://github.com/…git/</a>’: SSL certificate problem: self signed certificate in certificate chain</p>
<p>Any ideas?</p>
<p>Many thanks in advance for your support.</p>
<p>MtSixty7</p>

**Comment by MrSixty7** [comment, marked as answer]
<p>After a bit of research, I resolved this by turning off sll verifiction at the git cli, with command;</p>
<pre><code>gitconfig --global http.sslVerifyfalse
</code></pre>
<p>I then managed to clone the repo’s from GitHub and I have now reset it to true.</p>
<p>/MrSixty7 </p>

**Comment by AndreaGriffiths11** [comment]
<p>Hi <a class="mention" href="/mrsixty7">@mrsixty7</a>,</p>
<p>Thank you for sharing your solution! We’re looking forward to seeing you around!</p>
<p>Best,</p>
<p>Andrea</p>
